classes/UserReputation.php

```diff
@@ -247,6 +255,14 @@ public function __construct(&$player_rep, &$player_weekly_rep, &$player_pvp_rep,
      * Returns amount of reputation awarded for display/data confirmation purposes
      */
     public function addRep(int $amount, bool $bypass_weekly_cap = false, bool $increment_pvp = false): int {
+        // Double repuation
+        if (!empty($this->event) && $this->event instanceof DoubleReputation) {
+            $amount *= DoubleReputation::rep_modifier;
+        }
+
+        //Bonus seal reputation, 0 by default no need to check, bonus rep amounts are set by seal loading
+        $amount += $this->bonus_rep;
```

Comment:
This seems like it would apply to PvP too, right? Per last night I think we want to exclude PvP from these gains

Reply:
That's correct. Just hadn't had a chance to factor it in.